src/ddmd/mars.d

```diff
@@ -186,67 +187,6 @@ Where:
 ", FileName.canonicalName(global.inifilename), fpic, m32mscoff, mscrtlib);
 }
 
-/// DMD-generated module `__entrypoint` where the C main resides
-extern (C++) __gshared Module entrypoint = null;
-/// Module in which the D main is
-extern (C++) __gshared Module rootHasMain = null;
```

Comment:
I know. That was attempted 2 years ago but [didn't get pass Walter](https://github.com/dlang/dmd/pull/5282#r45561126).